barretenberg/cpp/src/barretenberg/vm/avm_trace/avm_trace.hpp

```diff
@@ -157,7 +172,70 @@ class AvmTraceBuilder {
     AvmKernelTraceBuilder kernel_trace_builder;
     AvmConversionTraceBuilder conversion_trace_builder;
 
+    /**
+     * @brief Create a kernel lookup opcode object
+     *
+     * Used for looking up into the kernel inputs (context) - {caller, address, etc.}
+     *
+     * @param dst_offset
```

Comment:
Please provide some description of the parameters even this sounds trivial for some of them.